www/status.php

```diff
@@ -27,10 +21,12 @@
     $shownServices = $services;
 } else {
     foreach ($services as $service) {
+        $serviceHost = $service['host'];
         $serviceName = $service['name'];
-        if (in_array($serviceName, array_keys($shownServicesList))) {
-            $service['name'] = $shownServicesList[$serviceName]['name'];
-            $service['description'] = $shownServicesList[$serviceName]['description'];
+        if (in_array($serviceHost, array_keys($shownServicesList)) &&
+            in_array($serviceName, array_keys($shownServicesList[$serviceHost]))) {
+            $service['name'] = $shownServicesList[$serviceHost][$serviceName]['name'];
+            $service['description'] = $shownServicesList[$serviceHost][$serviceName]['description'];
             array_push($shownServices, $service);
```

Comment:
I think, that this is too complicated.
It should be better to iterate over shownServices from configuration and read data from status.